corehq/apps/app_manager/tasks.py

```diff
@@ -1,9 +1,23 @@
 from celery.task import task
 from corehq.apps.users.models import CommCareUser
+from corehq.util.decorators import serial_task
 
 
 @task(queue='background_queue', ignore_result=True)
 def create_user_cases(domain_name):
     from corehq.apps.callcenter.utils import sync_usercase
     for user in CommCareUser.by_domain(domain_name):
         sync_usercase(user)
+
+
+@serial_task('{app._id}', max_retries=0)
+def make_async_build(app):
+    latest_build = app.get_latest_app(released_only=False)
+    if latest_build.version == app.version:
+        return
+    errors = app.validate_app()
+    if not errors:
```

Comment:
I was thinking something like a banner at the top of the page when viewing the app that says something like "There are errors preventing the latest version of this app from building. Phones will not receive the latest  version until these errors are corrected: <list of errors>"

I don't think we would want to notify the error on mobile since we should be alerting the person building the app that their change won't be deployed.

I think some build errors already display at the top of the page automatically whether you try to build or not, but am not sure if some errors will only be present during an actual build attempt.